src/main/kotlin/net/maizegenetics/phgv2/cli/Phg.kt

```diff
@@ -17,7 +17,7 @@ class Phg : CliktCommand() {
         setupDebugLogging()
 
         context {
-            helpFormatter = { MordantHelpFormatter(it, showRequiredTag = true) }
+            helpFormatter = { MordantHelpFormatter(it, showRequiredTag = true, showDefaultValues = true) }
```

Comment:
This is a good idea.  What happens in things like map-kmers where some of the parameters already have Default is X messages in the help text?  You should probably go through and fix everything as part of this so its consistent.

Reply:
Ok, I removed the hardcoded default values